TemplatesVSIX/SDL Terminology Provider/SDL Terminology Provider.csproj

```diff
@@ -54,36 +55,36 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Sdl.Core.PluginFramework">
-      <HintPath>$(ProgramFiles)\SDL\SDL Trados Studio\Studio15\Sdl.Core.PluginFramework.dll</HintPath>
+      <HintPath>..\SDL Trados Studio\bin\Debug\Sdl.Core.PluginFramework.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="Sdl.FileTypeSupport.Framework.Core">
       <Private>True</Private>
-      <HintPath>$(ProgramFiles)\SDL\SDL Trados Studio\Studio15\Sdl.FileTypeSupport.Framework.Core.dll</HintPath>
+      <HintPath>..\SDL Trados Studio\bin\Debug\Sdl.FileTypeSupport.Framework.Core.dll</HintPath>
```

Comment:
please use the $(ProgramFiles) for path so that is not specific per developer machine.